packages/fields/src/fields/featured-image/index.ts

```diff
@@ -18,6 +18,7 @@ const featuredImageField: Field< BasePost > = {
 	Edit: FeaturedImageEdit,
 	render: FeaturedImageView,
 	enableSorting: false,
+	isMediaField: true,
```

Comment:
I know we have the mediaField nomenclature, but it feels like it's a `previewField` and maybe we should update name in `view` object too? 🤔

Reply:
Every new code I'm introducing is updated to refer preview. I guess in a side PR we can change the already existing API of mediaField / showMedia.